Leetcode/1347/Minimum_step_no_to_make_two_strings_anagram.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+  Leetcode Problem: 1347
+  Problem link: (https://leetcode.com/problems/minimum-number-of-steps-to-make-two-strings-anagram/)
+*/
+
+#include<iostream>
+#include<vector>
+
+class Solution {
+public:
+    int minSteps(std::string s, std::string t) {
+        // Declare two vectors as frequency array to count the number of characters of a string
+        std::vector<int>f1(26,0);
+        std::vector<int>f2(26,0);
+        int i=0;
+        // Count number of characters in string s and put the frequency of character of string s
+        // in the index value of f1
+        while(s[i] != 0){
```

Comment:
Why are we avoiding using a range-based for loop?

Reply:
Yeah you are right. I should use range based for loop.